src/control/server/instance_drpc_test.go

```diff
@@ -116,26 +116,26 @@ func TestIOServerInstance_DrespToRankResult(t *testing.T) {
 		expResult   *MemberResult
 	}{
 		"rank success": {
-			expResult: &MemberResult{Rank: dRank, Action: "test", State: MemberStateJoined},
+			expResult: &MemberResult{Rank: dRank, State: MemberStateJoined},
 		},
 		"rank failure": {
-			daosResp: &mgmtpb.DaosResp{Status: -1},
+			daosResp: &mgmtpb.DaosResp{Status: int32(drpc.DaosNoSpace)},
 			expResult: &MemberResult{
-				Rank: dRank, Action: "test", State: MemberStateErrored, Errored: true,
-				Msg: fmt.Sprintf("rank %d dRPC returned DER -1", dRank),
+				Rank: dRank, State: MemberStateErrored, Errored: true,
+				Msg: fmt.Sprintf("rank %d: DAOS error (-1007): DER_NOSPACE", dRank),
```

Comment:
The drpc status codes should stringify for you.

Reply:
addressing in https://jira.hpdd.intel.com/browse/DAOS-3413